src/cli.ts

```diff
@@ -2,27 +2,28 @@
 
 import commander from 'commander';
 
-import fs from 'fs';
-import REPL from 'repl';
+import * as fs from 'fs';
+import * as REPL from 'repl';
 
-import pkg from '../package.json';
+// this import messes up the package structure after running tsc
+// import * as pkg from '../package.json';
+import { connect } from './';
 import log from './log';
-import { connect } from '.';
+import XAPI from './xapi/index.js';
 
+const pkg = { version: '4.2.0' };
```

Comment:
This needs to be fixed somehow. We imported `package.json` before, but `tsc` was not the biggest fan.